lib/linters/blank_line_after_block.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+    name: 'blankLineAfterBlock',
+    nodeTypes: ['atrule', 'rule'],
+    message: 'All blocks should be followed by a blank line.',
+
+    lint: function blankLineAfterBlockLinter (config, node) {
+        // Ignore at-rules without a body
+        if (node.ruleWithoutBody || (node.type === 'atrule' && !node.nodes)) {
+            /**
+             * Hopefully node.ruleWithoutBody will be implemented on AtRule nodes
+             * in the future: https://github.com/webschik/postcss-less/issues/55
+             */
+            return;
+        }
+
+        /**
+         * If it's the first node, ignore it since we actually check
+         * if the node is preceded by en an empty line
```

Comment:
I knew I forgot about something! I'll see if it's possible to solve it without an extra check (but I doubt it, though to how PostCSS gives us the nodes).